yarn.lock

```diff
@@ -9650,7 +9650,6 @@ micromatch@^4.0.0, micromatch@^4.0.2:
 
 "midi@https://github.com/paulrosen/MIDI.js.git#abcjs":
   version "0.4.2"
-  uid e593ffef81a0350f99448e3ab8111957145ff6b2
```

Comment:
![](https://media1.tenor.com/images/323accb4d3c53c7202305f5f32225713/tenor.gif?itemid=11222953)

Reply:
I hate this

Reply:
It hates you too

Reply:
how can I fix this @mrdrogdrog ?

Reply:
just add this file again

Reply:
done